modules/gitops-argo-cd-round-robin-sharding-algorithm.adoc

```diff
@@ -0,0 +1,138 @@
+
+The introduction of alternative sharding algorithms allows for further customization based on specific use cases. You can select the algorithm that best aligns with your deployment needs, which results in greater flexibility and adaptability in diverse operational scenarios.
+
+.Prerequisites
+* You have installed the {gitops-title} Operator in your cluster.
+* You have access to the {OCP} web console.
+* You have access to the cluster with `cluster-admin` privileges.
+
+.Procedure
+
+You can enable the `round-robin` sharding algorithm by either using the {OCP} web console or the command-line interface.
+
+* To enable the `round-robin` algorithm using the {OCP} web console, complete the following steps:
+
+. In the *Administrator* perspective of the web console, go to *Operators* → *Installed Operators*.
+
+. Click *{gitops-title}* from the installed operators and go to the *Argo CD* tab.
+
+. Click the Argo CD instance where you want to enable the `round-robin` sharding algorithm, for example `openshift-gitops`.
+
+. Click the *YAML* tab and edit the yaml file:
++
+.Example Argo CD instance with round-robin sharding algorithm enabled
+[source,yaml]
+----
+apiVersion: argoproj.io/v1beta1
+kind: ArgoCD
+metadata:
+  name: openshift-gitops
+  namespace: openshift-gitops
+spec:
+  controller:
+    processors: {}
+    sharding:
+      enabled: true <1>
+      replicas: 3 <2>
+    env: <3>
+      - name: ARGOCD_CONTROLLER_SHARDING_ALGORITHM 
+        value: round-robin
+----
+<1> Set the `sharding.enabled` parameter to `true` to enable sharding.
+<2> Set the number of replicas to the desired value, for example `3`.
+<3> Set the sharding algorithm to `round-robin`.
+
+. Click *Save* to save the changes.
+
+* To enable the `round-robin` algorithm using the command-line interface, complete the following steps:
+
+. Edit the Argo CD instance YAML file to enable sharding:
++
+[source,terminal]
+----
+$ oc patch argocd <argocd_instance> -n <namespace> --patch='{"spec":{"controller":{"sharding":{"enabled":true}}}}' --type=merge
+----
++
+.Example output
+[source,terminal]
+----
+argocd.argoproj.io/<argocd_instance> patched
+----
+
+. Configure the sharding algorithm to `round-robin`:
++
+[source,terminal]
+----
+$ oc patch argocd <argocd_instance> -n <namespace> --patch='{"spec":{"controller":{"env":[{"name":"ARGOCD_CONTROLLER_SHARDING_ALGORITHM","value":"round-robin"}]}}}' --type=merge
+----
++
+.Example output
+[source,terminal]
+----
+argocd.argoproj.io/<argocd_instance> patched
+----
+
+. Set the number of replicas to the desired value:
++
+[source,terminal]
+----
+oc patch argocd <argocd_instance> -n <namespace> --patch='{"spec":{"controller":{"sharding":{"replicas":<value>}}}}' --type=merge
+----
++
+.Example output
+[source,terminal]
+----
+argocd.argoproj.io/<argocd_instance> patched
+----
+
+.Verification
+
+. Verify that the sharding is enabled with `round-robin` as the sharding algorithm:
```

Comment:
@ekristova Suggest weaving the verification steps as part of the steps in the procedure to ensure parallelism.